caampr-backend/src/main/java/com/jelistan/caampr/lambda/model/Profile.java

```diff
@@ -1,14 +1,20 @@
 package com.jelistan.caampr.lambda.model;
 
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
 import lombok.Builder;
 import lombok.Data;
 
+@DynamoDBTable(tableName = "Gear-v1")
```

Comment:
The profile model should live in its own table and not in the gear table.  So something like `Profile-v1`

Reply:
ah gotcha, thought they shared the table and the profile just served as a secondary index or something, will do